### PR TITLE
Pop-up Window

### PR DESCRIPTION
**Pull request by RitaDee**
### In this milestone, we implemented the following:

- Used JavaScript array to store all of the information for all projects.
- Updated the main page so that the projects section is created dynamically using the information stored in the JavaScript object. 
- Implemented the popup window for both the desktop and mobile versions.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -461,6 +461,123 @@ span {
...
+  width: 100%;
+  padding: 10px;
+  background-color: #979493;
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure your modal container has a background that is identical to the one in the provided Figma design.
 
    - **YOUR DESIGN**
      image

    - **FIGMA DESIGN**
    
      image

File: style.css [hunk 2]
```diff
@@ -461,6 +461,123 @@ span {
...
+  margin-left: 2px;
+  margin-top: 8px;
+  border: none;
+  padding: 13px;
+  color: white;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure your button contents are centered in order to meet the provided Figma design :wink:

    - **YOUR DESIGN**
     image

    - **FIGMA DESIGN**
   image

File: style.css [hunk 3]
```diff
@@ -461,6 +461,123 @@ span {
...
+
+.modal-tech {
+  width: 100%;
+}
+
+.modal-tech ul {
+  display: flex;
+  justify-content: space-between;
+  width: 75%;
+}
```
Comment (levy002) [inline style.css:562]: - [ ] Kindly make sure to provide enough space between the stack technologies I order to look like the provided Figma design. :wink:

     - **YOUR DESIGN**
     image

     - **FIGMA DESIGN**
     image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @RitaDee   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 
- Excellent UI :heavy_check_mark: 
- All linters are green :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure on the desktop version user can be able to see the `See Project` button maybe by hovering over the project card. At this moment we are not able to other rest five card details pop-up. Kindly fix that. :wink: 

    - **YOUR APP BEHAVIOUR**
    
     chrome-capture-2023-0-5 (1)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure to add all contributors in the `Authors` section to your README file. :wink: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -587,6 +589,12 @@ span {
     display: none;
   }
 
+  /* .unique-btn {
+    position: absolute;
+    bottom: -113px;
+    width: 49.1%;
+  } */
+
```
Comment (GedeonTS) [inline style.css]: - [ ] Please make sure to remove these commented-out lines of code you just added to make your project cleaner :recycle:

File: style.css [hunk 5]
```diff
@@ -814,6 +827,21 @@ span {
...
+    font-weight: 700;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    height: 48px;
+    color: #fff;
+    width: 100%;
+    border: none;
+  }
```
Comment (GedeonTS) [inline style.css]: - [ ] Good job adding the see project button on hover effect as suggest by the previous reviewer [here](https://github.com/RitaDee/portfolio/pull/20#:~:text=Kindly%20make%20sure%20on%20the%20desktop%20version%20user%20can%20be%20able%20to%20see%20the%20See%20Project%20button%20maybe%20by%20hovering%20over%20the%20project%20card.%20At%20this%20moment%20we%20are%20not%20able%20to%20other%20rest%20five%20card) :clap: , however, while doing it you have changed your design, please make it look back like the Figma design :record_button: 

Your current design:
YourStacks

Your previous & figma:
figstacks

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @RitaDee , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights :tophat: :+1: 
- Good use of Github flow :+1: 
- No linter errors :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @RitaDee 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**